packages/clay-core/src/table/Row.tsx

```diff
@@ -29,24 +31,78 @@ type Props<T> = {
 	 * Property to render content with dynamic data.
 	 */
 	items?: Array<T>;
+
+	/**
+	 * Internal prop.
+	 * @ignore
+	 */
+	keyValue?: React.Key;
+
+	/**
+	 * Internal prop.
+	 * posinset
+	 * @ignore
+	 */
+	_index?: number;
```

Comment:
Why are some of these internal props prefixed with an underscore but `keyValue` isn't?

Reply:
`keyValue` and `index` are properties coming from collection, while the properties `_index`, `_size`... are properties coming from flatten when we reassemble the items to a list, so they are just to differentiate where these values are coming from.